plugins/action_update_review_status/src/updateReviewsStatuses.ts

```diff
@@ -99,12 +121,16 @@ const updateReviewsStatuses: ActionPluginType = async ({
       }
     } else {
       // For Application submission
+
+      // Get reviews matching application_id, current stage, in level 1
       const level = 1
-      // Get reviews/review assignments (with status) matching application_id & current stage
       const reviews = await getReviewsByLevelAndStatus(level)
       // Deduce which ones should be updated to Pending
       for (const review of reviews) {
         const { reviewAssignmentId, reviewStatus } = review
+        if (reviewStatus === ReviewStatus.Locked) {
+          // TODO: Check with team should also lock the reviewAssignment ?
```

Comment:
Did you get an answer on this, and do you want to figure it out before merging this PR, or wait till later?

Reply:
No... I'll leave this one there until we decide. Will add a discussion issue to check that.

Reply:
Issue #757